### ISSUE TITLE
[MNG-6987] Reorder groupId before artifactId when writing an exclusion using maven-model

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#8047]
**[Marc Bruggmann](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=marcbr)** opened **[MNG-6987](https://issues.apache.org/jira/browse/MNG-6987?redirect=false)** and commented

We are using `maven-model` to parse, modify, and write back a POM file roughly like so:

 

```java
MavenXpp3Reader reader = new MavenXpp3Reader(); 
Model model = reader.read(new FileReader(input)); 
// make modifications to the model 
MavenXpp3Writer writer = new MavenXpp3Writer(); 
writer.write(new FileWriter(output), model);
```

 

The exclusion shows up in the output file like this:

```java
<exclusions>
  <exclusion>
    <artifactId>plexus-cipher</artifactId>
    <groupId>org.sonatype.plexus</groupId>
  </exclusion>
</exclusions>
```

In most other places in the POM, we order the groupId before the artifactId. It would be nice to do it the same way for exclusion, like so:

```java
<exclusions>
  <exclusion>
    <groupId>org.sonatype.plexus</groupId>
    <artifactId>plexus-cipher</artifactId>
  </exclusion>
</exclusions>
```

---

**Affects:** 3.6.3

**Remote Links:**
- [GitHub Pull Request #375
](https://github.com/apache/maven/pull/375)

**Backported to:** [4.0.0-alpha-2](https://github.com/apache/maven/milestone/113?closed=1), [3.8.2](https://github.com/apache/maven/milestone/103?closed=1)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with 8b7f6f0c9d15be7b90c4e40bfdd296ce4be01f57.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [a27b4ee595ee48833fa659ee30ff463f26c4a50d](https://gitbox.apache.org/repos/asf?p=maven.git;a=commit;h=a27b4ee595ee48833fa659ee30ff463f26c4a50d) for `maven-3.8.x` branch.
